<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajKodu</class>
 <widget class="QDialog" name="szukajKodu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="kodLE"/>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="odCenyDSB"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="nazwaCB">
      <property name="text">
       <string>Nazwa</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="odCenyCB">
      <property name="text">
       <string>Od ceny za km</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doCenyDSB"/>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="doCenyCB">
      <property name="text">
       <string>Do ceny za km</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="nazwaLE"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="kodCB">
      <property name="text">
       <string>Kod</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
